cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -477,7 +477,7 @@ def _verify_deprecated_info(self, course_id, advanced_modules, info, deprecated_
         self.assertEqual(info['block_types'], deprecated_block_types)
         self.assertEqual(
             info['block_types_enabled'],
-            any(component in advanced_modules for component in deprecated_block_types)
+            [component for component in advanced_modules if component in deprecated_block_types]
```

Comment:
> Would it make more sense to have a test that verifies that the components are all deprecated

It would, but that integration-level testing is far beyond the scope of this unit test IMO.

> another test that verifies that we aren't showing components that the course isn't using?

Added